### PR TITLE
doc: add cluster sizing examples

### PR DESCRIPTION
**Pull request by schoren**
This PR adds example value files with proposals on how to size onprem clusters

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,6 +218,27 @@ global:
 ```
 </details>
 
+## Cluster sizing
+
+Depending on the size of your organization, your deployment might require different sizes. We provide a `small` and `large` deployment sizing examples.
+To use it you can:
```
Comment (jfermi) [inline README.md:224]:
```suggestion
Use the following commands to select the size for your deployment:
```

File: README.md [hunk 2]
```diff
@@ -218,6 +218,27 @@ global:
 ```
 </details>
 
+## Cluster sizing
```
Comment (jfermi) [inline README.md:221]:
```suggestion
## Cluster Sizing
```

**Review by jfermi — COMMENTED** [review]
A couple of minor editing suggestions.

**Review by mathnogueira — APPROVED** [review]

**Review by xoscar — APPROVED** [review]
GJ @schoren

**Pull request merged**